src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/ActivityHelper.cs

```diff
@@ -20,6 +20,7 @@ internal static class ActivityHelper
     /// </summary>
     internal const string ContextKey = "__AspnetInstrumentationContext__";
     internal static readonly object StartedButNotSampledObj = new();
+    internal static readonly RequestDataHelper RequestDataHelper = new(configureByHttpKnownMethodsEnvironmentalVariable: false);
```

Comment:
Why false? This package in other places is using true as parameter.

```suggestion
    internal static readonly RequestDataHelper RequestDataHelper = new(configureByHttpKnownMethodsEnvironmentalVariable: true);
```

Reply:
I took TelemetryHelper within AspNetCore instrumentation for example, in which it is false.
But with further reading of the parameter it seems like passing true is the correct way to go.

Reply:
It is also false in OpenTelemetry.Instrumentation.Http

Reply:
You are right, there is no possibility to configure it in AspNetCore. As I remember it is not supported by .NET8+ with native instrumentations.

Reply:
Okay, since I already changed it to true I will revert it :)

Reply:
Please revert your revert. It is not supported on .NET8 for AspNetCore.
AspNet is something different, ans the parameter should be true!